GeosCore/flexgrid_read_mod.F90

```diff
@@ -292,7 +293,18 @@ SUBROUTINE FlexGrid_Read_A1( YYYYMMDD, HHMMSS, Input_Opt, State_Grid, &
     ! Read FRSNO
     v_name = "FRSNO"
     CALL Get_Met_2D( Input_Opt, State_Grid, Q, TRIM(v_name), t_index=t_index )
-    State_Met%FRSNO = Q
+
+    ! The import FRSNO is fraction of land with snow cover. Convert to
```

Comment:
Change FRSNO to FRSNOW

Reply:
FRSNO is still used in the file and is only renamed for State_Met.